selfdrive/hardware/base.h

```diff
@@ -1,25 +1,50 @@
 #pragma once
 
 #include <cstdlib>
-#include <fstream>
+#include <string>
+
+#include "selfdrive/common/mat.h"
 
 // no-op base hw class
-class HardwareNone {
+class HardwareBase {
 public:
-  static constexpr float MAX_VOLUME = 0;
-  static constexpr float MIN_VOLUME = 0;
+  float MAX_VOLUME = 0;
+  float MIN_VOLUME = 0;
+  int road_cam_focal_len = 910;
```

Comment:
think it’s better for this to live in camerad

Reply:
I think for now it's fine to be here. Ideally it lives in camerad, but it is then communicated over visionipc when you get the frames, in that case the subscriber doesn't have to know which hardware it's running on.